src/components/agent/implementation/web-crypto-api.ts

```diff
@@ -0,0 +1,74 @@
+import { DIDKey } from "iso-did/key"
+import { spki } from "iso-signatures/spki"
+import * as crypto from "../../../common/crypto.js"
+import { Store } from "../../../common/crypto/store.js"
+import { hasProp, isObject } from "../../../common/index.js"
```

Comment:
```suggestion
import { isObject } from "../../../common/index.js"
```

Looks like `hasProp` is unused.